### DISCUSSION TITLE
How to select a ReqIF definition

### DISCUSSION THREAD

**Discussion by reqTibi** [IBM/ELM-Python-Client#36] [General]
I have a question: how to specify for reqif_io a certain ReqIF definition, if there more than one, for export. Currently all definitions will be exported.

**Comment by barny** [comment]
If you don't specify a name then all definitions in the project are exported. If you want one or more specific definitions exported provide their name(s) in the exact capitalisation separated by spaces - if the name includes space then put " around the name.

It's possible to specify a regular expression pattern for the name and it will export all that match. A Regular expression has any of ^$?*[].+() in it and unless you use ^ or $ will match anywhere in the name. If you want to find out more do some goggling for regular expressions, and the code is around lines 274-299 in the current reqif_io.py.

It's only an example, you've got the source code can adapt/extend it to meet your needs.

**Reply by reqTibi** [reply]
Thank you very much for you support, helps me a lot.
